ui/app/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js

```diff
@@ -209,10 +219,12 @@ export default class ConfirmApproveContent extends Component {
                     showEdit: true,
                     onEditClick: () => showEditApprovalPermissionModal({
                       customTokenAmount,
+                      decimals,
+                      origin,
```

Comment:
With this fix, maybe we should also make `origin` required in the `editApprovalPermissions` component?

Reply:
Yes, that's an excellent point, I will do that. I hadn't considered this because I fixed this in the first place accidentally 😅 then noticed looking back at the diff.

Reply:
Fixed here: https://github.com/MetaMask/metamask-extension/pull/7920/commits/b723802f3ec71dd69fb4e341da64d89c186ce319